projects/bmp/.gitignore

```
@@ -0,0 +1,5 @@
+.pio
+.vscode/.browse.c_cpp.db*
+.vscode/c_cpp_properties.json
+.vscode/launch.json
+.vscode/ipch
```

Wydaje mi się że cały folder `.vscode` jest do gitignore

```suggestion
.pio/
.vscode/
```

To oznacza że będzie trzeba usunąć z gita już trackowane pliki:

```bash
git rm -r --cached .vscode
```